setup/security/ssl.md

```diff
@@ -313,27 +367,42 @@ If you have problems...
 * If you are running Spinnaker in a distributed environment, have you run
 `hal deploy connect`?
 
-## About network configurations
-
-Each authentication mechanism is configured differently depending on where the
-SSL connection terminates.
-
-### Server-terminated SSL
 
-Terminating SSL within the Gate server is the de-facto way to enable SSL for
-Spinnaker. This works with or without a load balancer proxying traffic to this
-instance.
+## Using a custom CA for internal communications
+There are a lot of places in Spinnaker which support the ability to configure custom Java trust/key stores for 
+organizations who use internally signed certificates. In some cases, however, this isn’t supported yet but you still 
+need to talk to a service which serves one of these certificates. This post will show you how to import your 
+certificate into a Java trust/key store and configure a Spinnaker service with it.
 
-![SSL terminated at server through load balancer](/setup/security/authentication/network-arch/server-ssl-termination.png)
+Create a temporary copy of your system’s Java trust/key store and import your internal certificate. If you’re on a Mac,
+this will be located at `/usr/libexec/java_home/)/jre/lib/security/cacerts`. It will be different on Linux.
```

Comment:
Is there any general place one might look for this on Linux? (I have zero or near-zero experience with doing anything related to this on Linux.)

Reply:
Sadly, there's a LOT of variations on linux/containers.  I could list some of examples, but most linux people should be aware of where they've installed it...  or one would hope.